resdk/scripts.py

```diff
@@ -350,3 +352,122 @@ def upload_reads():
+
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                     description=description)
+
+    parser.add_argument('-a', '--address', default='https://torta.bcm.genialis.com',
+                        help='Resolwe server address')
+    parser.add_argument('-e', '--email', default='admin', help='User name')
+    parser.add_argument('-p', '--password', default='admin', help='User password')
+    parser.add_argument('-v', '--verbose', action='store_true', help='Enable verbose reporting')
+    parser.add_argument('features', help='ZIP file containing feature descriptions')
+
+    args = parser.parse_args()
+
+    if args.verbose:
+        resdk_logger.start_logging()
+
+    resolwe = Resolwe(args.email, args.password, args.address)
+
+    subtype_map = {
+        'processed_pseudogene': 'pseudo',
+        'unprocessed_pseudogene': 'pseudo',
+        'polymorphic_pseudogene': 'pseudo',
+        'transcribed_unprocessed_pseudogene': 'pseudo',
+        'unitary_pseudogene': 'pseudo',
+        'transcribed_processed_pseudogene': 'pseudo',
+        'transcribed_unitary_pseudogene': 'pseudo',
+        'TR_J_pseudogene': 'pseudo',
+        'IG_pseudogene': 'pseudo',
+        'IG_D_pseudogene': 'pseudo',
+        'IG_C_pseudogene': 'pseudo',
+        'TR_V_pseudogene': 'pseudo',
+        'IG_V_pseudogene': 'pseudo',
+        'pseudogene': 'pseudo',
+        'pseudo': 'pseudo',
+        'asRNA': 'asRNA',
+        'antisense': 'asRNA',
+        'protein_coding': 'protein-coding',
+        'protein-coding': 'protein-coding',
+        'IG_V_gene': 'protein-coding',
+        'IG_LV_gene': 'protein-coding',
+        'TR_C_gene': 'protein-coding',
+        'TR_V_gene': 'protein-coding',
+        'TR_J_gene': 'protein-coding',
+        'IG_J_gene': 'protein-coding',
+        'TR_D_gene': 'protein-coding',
+        'IG_C_gene': 'protein-coding',
+        'IG_D_gene': 'protein-coding',
+        'miRNA': 'ncRNA',
+        'lincRNA': 'ncRNA',
+        'processed_transcript': 'ncRNA',
+        'sense_intronic': 'ncRNA',
+        'sense_overlapping': 'ncRNA',
+        'bidirectional_promoter_lncRNA': 'ncRNA',
+        'ribozyme': 'ncRNA',
+        'Mt_tRNA': 'ncRNA',
+        'Mt_rRNA': 'ncRNA',
+        'misc_RNA': 'ncRNA',
+        'macro_lncRNA': 'ncRNA',
+        '3prime_overlapping_ncRNA': 'ncRNA',
+        'sRNA': 'ncRNA',
+        'snRNA': 'snRNA',
+        'scaRNA': 'snoRNA',
+        'snoRNA': 'snoRNA',
+        'rRNA': 'rRNA',
+        'ncRNA': 'ncRNA',
+        'tRNA': 'tRNA',
+        'other': 'other',
+        'unknown': 'unknown'
+    }
+
+    errors = 0
+    updated = 0
+    with zipfile.ZipFile(args.features) as archive:
+        for entry in archive.infolist():
+            if not entry.filename.endswith('.tab'):
+                continue
+
+            if args.verbose:
+                print('Importing features from "{}"...'.format(entry.filename))
```

Comment:
Since `verbose` starts logging, it would probably make sense to replace `print` statements with logger calls?